commit d5eedd87d34d3093e958809c9feef08b3651aa06
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Jun 20 09:55:43 2018 +0100

    BCM270X_DT: Remove AUX interrupt controller refs
    
    As of 4.18 we've dropped the auxilliary interrupt controller because
    the upstream code has a fix for the SPI driver (effectiveness TBD).
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm270x.dtsi b/arch/arm/boot/dts/bcm270x.dtsi
index d26b10ad0245..145e455a2a14 100644
--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -147,29 +147,6 @@
 	};
 };
 
-/* Configure and use the auxilliary interrupt controller */
-
-&aux {
-	interrupts = <1 29>;
-	interrupt-controller;
-	#interrupt-cells = <1>;
-};
-
-&uart1 {
-	interrupt-parent = <&aux>;
-	interrupts = <0>;
-};
-
-&spi1 {
-	interrupt-parent = <&aux>;
-	interrupts = <1>;
-};
-
-&spi2 {
-	interrupt-parent = <&aux>;
-	interrupts = <2>;
-};
-
 &vc4 {
 	status = "disabled";
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1501_d5eedd87d34d3093e958809c9feef08b3651aa06.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1501_d5eedd87d34d3093e958809c9feef08b3651aa06.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
